x-pack/filebeat/input/azureeventhub/eph.go

```diff
@@ -1,114 +0,0 @@
-// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
-// or more contributor license agreements. Licensed under the Elastic License;
-// you may not use this file except in compliance with the Elastic License.
-
-//go:build !aix
-
-package azureeventhub
-
-import (
-	"context"
-	"errors"
-	"fmt"
-
-	eventhub "github.com/Azure/azure-event-hubs-go/v3"
-	"github.com/Azure/azure-event-hubs-go/v3/eph"
-	"github.com/Azure/azure-event-hubs-go/v3/storage"
-	"github.com/Azure/azure-storage-blob-go/azblob"
-	"github.com/Azure/go-autorest/autorest/azure"
-)
-
-// users can select from one of the already defined azure cloud envs
-var environments = map[string]azure.Environment{
-	azure.ChinaCloud.ResourceManagerEndpoint:        azure.ChinaCloud,
-	azure.GermanCloud.ResourceManagerEndpoint:       azure.GermanCloud,
-	azure.PublicCloud.ResourceManagerEndpoint:       azure.PublicCloud,
-	azure.USGovernmentCloud.ResourceManagerEndpoint: azure.USGovernmentCloud,
-}
-
-// runWithEPH will consume ingested events using the Event Processor Host (EPH).
-//
-// To learn more, check the following resources:
-// - https://github.com/Azure/azure-event-hubs-go#event-processor-host
-// - https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-event-processor-host
-func (a *azureInput) runWithEPH() error {
```

Comment:
Mainly curious, is there any reason why you moved this code to `v1_input.go`? Is it just related to the legacy input and it's preparation for adding the modern one?